zebra-state/src/service/memory_state/chain.rs

```diff
@@ -511,6 +484,25 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn bigger_is_greater() -> Result<()> {
```

Comment:
I'm not sure exactly what this test is trying to check

Reply:
In particular, longer side-chains can have less work, so bigger chains are not always greater.

But this test is checking that adding additional blocks to the *same* chain adds more work, which is a useful property.

It might be helpful if we tested adding different work values - this test only adds a copy of the parent block's work.
It might also be helpful to test on every block in `zebra_test::vectors::BLOCKS`.

Reply:
its checking that the longer chain always has a greater difficulty

Reply:
That property isn't guaranteed - the best chain can be shorter than a side-chain

Reply:
Can you update the test function name or add a doc comment to clarify?